### PR TITLE
Bump app version in Helm Chart from 0.19.0 to 0.22.0

### PR DESCRIPTION
**Pull request by isandeep41**
Upgrade app version to support 0.22.0 build

This PR has:
- [x] been self-reviewed.
   - [ ] using the [concurrency checklist](https://github.com/apache/druid/blob/master/dev/code-review/concurrency.md) (Remove this item if the PR doesn't have any relation to concurrency.)
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. Linked related entities via Javadoc links.
- [ ] added or updated version, license, or notice information in [licenses.yaml](https://github.com/apache/druid/blob/master/dev/license.md)
- [x] added comments explaining the "why" and the intent of the code wherever would not be obvious for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold for [code coverage](https://github.com/apache/druid/blob/master/dev/code-review/code-coverage.md) is met.
- [ ] added integration tests.
- [x] been tested in a test Druid cluster.

### CODE REVIEW

File: helm/druid/Chart.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 apiVersion: v2
-appVersion: 0.19.0
+appVersion: 0.22.0
 description: DEPRECATED Apache Druid is a high performance real-time analytics database.
```
Comment (FrankChen021) [inline helm/druid/Chart.yaml:18]: What does the 'DEPRECATED' mean ?

Comment (isandeep41) [reply helm/druid/Chart.yaml:18]: The Druid Chart was deprecated in [helm/charts](https://github.com/helm/charts/tree/master/incubator/druid) repo
> ⚠️ Repo Archive Notice
> As of Nov 13, 2020, charts in this repo will no longer be updated. For more information, see the Helm Charts Deprecation and Archive Notice, and Update.

The deprecation notice can be removed from this repo. @asdf2014 what do you suggest ?

Comment (FrankChen021) [reply helm/druid/Chart.yaml:18]: I think we can remove this flag.

Comment (isandeep41) [reply helm/druid/Chart.yaml:18]: the flag and message has been removed in #11897

**Review by FrankChen021 — APPROVED** [review]

**Review by asdf2014 — APPROVED** [review]
LGTM :+1:

**Pull request merged**